src/main/java/frc/robot/subsystems/Lift.java

```diff
@@ -303,7 +315,16 @@ public void initialize() {
         elevatorDutyCycleEncodersDifferenceDegrees
                 * Constants.Lift.ELEVATOR_MOTOR_ENCODER_DIFFERENCES_SCALAR_INCHES_PER_DEGREE
             - Cal.Lift.ELEVATOR_ABS_ENCODER_POS_AT_START_INCHES);
-  }
+
+            armController.setTolerance(Cal.Lift.ARM_ALLOWED_CLOSED_LOOP_ERROR_DEG);
+            // armController.enableContinuousInput(0.0, 360.0);
+            armController.disableContinuousInput();
+            armController.reset(armEncoder.getPosition());
+            armController.setGoal(armEncoder.getPosition());
```

Comment:
These changes are important. And let's do this setGoal to current position for the elevator and intake too